Update streamlit_guide.rst (#93)
recognai/rubrix: docs/guides/streamlit_guide.rst

Monitoring and collecting data from third-party apps
====================================================

> This guide will show you **how can Rubrix be integrated into third-party applications** to collect predictions and user feedback. For this one we are going to use `streamlit <https://streamlit.io>`_\ , an amazing tool that turns Python scripts into beautiful web-apps. 

Author: ```suggestion
This guide will show you **how can Rubrix be integrated into third-party applications** to collect predictions and user feedback. To do this, we are going to use `streamlit <https://streamlit.io>`_\ , an amazing tool to turn Python scripts into beautiful web-apps. 
```
